nsqd/tcp.go

```diff
@@ -31,7 +31,7 @@ func (p *tcpServer) Handle(clientConn net.Conn) {
 
 	var prot protocol.Protocol
 	switch protocolMagic {
-	case "  V2":
+	case "V2":
```

Comment:
This line has been here since 2013. See also: https://github.com/nsqio/go-nsq/blob/master/protocol.go#L14